lib/internal/fs/promises.js

```diff
@@ -526,17 +524,14 @@ async function readFileHandle(filehandle, options) {
 
   let size = 0;
   let length = 0;
-  if ((statFields[1/* mode */] & S_IFMT) === S_IFREG) {
-    size = statFields[8/* size */];
+  if ((statFields[1 /* mode */] & S_IFMT) === S_IFREG) {
+    size = statFields[8 /* size */];
```

Comment:
nit: I've noticed that many changes are related to linting.
I would suggest reverting them to make the changes introduced by this PR more clear

Reply:
you are right, thank you

Reply:
ı updated